styles.css

```diff
@@ -1,826 +1,980 @@
+    flex-direction: column;
+    align-items: flex-start;
+  }
+
+  .container-span {
+    background: #f7f7f9;
+    border-radius: 8px;
+    width: 303px;
+    height: auto;
+    padding: 0;
+    margin: 6px;
+  }
+
+  .icone {
+    display: flex;
+    align-items: center;
+  }
+
+  img.fav {
+    margin-right: 16px;
+  }
+
+  li.frameworks,
+  li.skills {
+    border-bottom: 1px solid #f7f7f9;
+  }
+
+  .frameworks {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .skills {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  /* ---------Form Section------------ */
+
+  form {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .contact-form {
+    margin-top: 10rem;
+    background-color: #6070ff;
+    background-repeat: no-repeat;
+    background-position-x: 350px;
+    padding: 80px 10px 10px;
+    border-top-left-radius: 120px;
+    text-align: center;
+  }
+
+  .contact-content h1 {
+    font-family: Poppins, sans-serif;
+    color: #fff;
+    text-align: center;
+    height: 56px;
+    font-size: 40px;
+    line-height: 56px;
+    margin: 20px 0;
+  }
+
+  .name-container input {
+    font-family: Poppins, sans-serif;
+    border: 1px solid transparent;
+    border-radius: 8px;
+    background: #fff;
+    font-size: 17px;
+    line-height: 20px;
+    color: #172b4d;
+    width: 80%;
+    margin-top: 20px;
+    height: 48px;
+    padding-left: 5px;
+  }
+
+  .mail-container input {
+    font-family: Poppins, sans-serif;
+    border: 1px solid transparent;
+    border-radius: 8px;
+    background: #fff;
+    font-size: 17px;
+    line-height: 20px;
+    color: #172b4d;
+    width: 80%;
+    margin-top: 20px;
+    height: 48px;
+    padding-left: 5px;
+  }
+
+  .textarea-container textarea {
+    font-family: Poppins, sans-serif;
+    border: 1px solid transparent;
+    border-radius: 8px;
+    background: #fff;
+    font-size: 17px;
+    line-height: 20px;
+    color: #172b4d;
+    width: 80%;
+    margin-top: 20px;
+    height: 170px;
+    padding-left: 5px;
+  }
+
+  button {
+    width: 140px;
+    height: 48px;
+    margin-left: 40px;
+    border: 1px solid #6070ff;
+    font-family: Poppins, sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    background: #fff;
+    border-radius: 8px;
+    color: #6070ff;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+  }
+}
+
+/* ----------------Desktop Version------ */
+
+@media only screen and (min-width: 768px) {
+  header {
+    display: flex;
+    justify-content: space-between;
+    margin: 9px;
+    background-color: #fff;
+    box-shadow: 0 1px 4px lightgrey;
+    align-items: center;
+    width: 100%;
+  }
+
+  nav {
+    display: block;
+    margin: 16px 142px 16px 0;
+  }
+
+  nav a.links {
+    margin-right: 12px;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
```

Comment:
- Please, kindly ensure your list items are well aligned to match the design as much as you can below:

![nav](https://user-images.githubusercontent.com/59397159/187049544-312d1050-9d40-4ba9-b4f7-c3b5bb6c7d6c.png)